packages/embark-proxy/src/index.ts

```diff
@@ -61,7 +61,7 @@ export default class ProxyManager {
     });
   }
 
-  private async setupProxy() {
+  private async setupProxy(clientName: string, node?: string) {
```

Comment:
Why do we need to pass `clientName` in to this method?

Reply:
Additionally, if we go with my suggestion, we don't need to pass `node` in either.